src/ui/public/kbn_top_nav/bread_crumbs/bread_crumbs.js

```diff
@@ -8,13 +8,15 @@ const module = uiModules.get('kibana');
 module.directive('breadCrumbs', function ($location) {
   return {
     restrict: 'E',
+    replace: true,
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1238659/22576828/e53cd212-e972-11e6-8711-fc5e2776ab60.png)

It will remove the `<bread-crumbs>` element from the DOM and instead only add the children (i.e. the DOM defined by the directive's template).